frontend/src/hooks/useKeycloakWrapper.tsx

```diff
@@ -1,60 +1,140 @@
 import { useKeycloak } from '@react-keycloak/web';
-import { SYSTEM_ADMINISTRATOR, AGENCY_ADMINISTRATOR } from 'constants/strings';
+import { Roles } from 'constants/roles';
 
+/**
+ * IUserInfo interface, represents the userinfo provided by keycloak.
+ */
 export interface IUserInfo {
+  displayName?: string;
   username: string;
   name?: string;
   preferred_username?: string;
   firstName?: string;
   lastName?: string;
   email: string;
-  Groups: any[];
+  groups: string[];
```

Comment:
What comes out of the `userinfo` endpoint is actually "Groups" (case-sensitive). If you are going to rename in the frontend, also provide instructions on how to change the Groups property in Keycloak

Reply:
This is configured by Keycloak and it will be 'groups'

Reply:
well , then please also export the current configuration or make scripts that update our local environments. The current config on my local is Groups

Reply:
Ah yes I need to update the local keycloak!

Reply:
The realm export in source control is way outdated. Before making further changes that break everyone else's setup please update the realm.json here